src/OSPSuite.Core/Domain/Services/EntityValidator.cs

```diff
@@ -78,24 +76,13 @@ public void Visit(IEntity entity)
 
       public void Visit(IParameter parameter)
       {
-         //only validate visible parmaeter
+         //only validate visible parameter
          if (!parameter.Visible)
             return;
 
-         //do not validate parameter defined in molecule amount for other molecule than drug (value cannot be set by user and some values are invalid)
-         if (parentIsAMoleculeButNotDrug(parameter.ParentContainer))
```

Comment:
@Yuri05 Becasue we are removing local nan parameters now, I don't think this is required anymore. This will also ensure that we validate all parameters of a enzyme that may be invalid